tests/app/data/test_schema_validation.py

```diff
@@ -83,8 +75,6 @@ def validate_json_against_schema(file, json_to_validate, schema):
             return []
         except jsonschema.exceptions.ValidationError as e:
             return ["Schema Validation Error! File [{}] does not validate against schema. Error [{}]".format(file, e)]
-        except JSONDecodeError as e:
```

Comment:
it will but the functionality before displayed the errors with all the schemas on the 1st run. if all the schemas are broken this will currently only show you the 1st